install_config/http_proxies.adoc

```diff
@@ -192,10 +192,18 @@ leaving the external HTTP proxy as the only option.
 ----
 HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
 HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
-NO_PROXY=master.hostname.example.com,172.30.123.45,docker-registry.default.svc.cluster.local
+NO_PROXY=master.hostname.example.com,<docker-registry-ip>,docker-registry.default.svc.cluster.local
 ----
 ====
 
+To retrieve the <docker-registry-ip> run the following commands:
+
+----
+# oc project default
+# oc rsh <docker-registry-pod>
+# printenv DOCKER_REGISTRY_SERVICE_HOST
+----
```

Comment:
I think this step could be simplified to just running `oc describe svc/docker-registry` and checking `IP`?  If so, it could also just be a footnote at the end of the above `NO_PROXY=` line, or as its own step before the "Edit the /etc/sysconfig/docker file" step.